app/api/attendees.py

```diff
@@ -122,7 +129,9 @@ def query(self, view_kwargs):
                 raise ForbiddenException({'source': ''}, 'Access Forbidden')
             query_ = query_.join(User, User.email == TicketHolder.email).filter(User.id == user.id)
 
-        query_ = event_query(self, query_, view_kwargs, permission='is_registrar')
+        if view_kwargs.get('event_id'):
```

Comment:
Why is this needed?

Reply:
Because we're running it irrespective of we have event_id in view_kwargs or not
Though its checked inside the function, but I made it so that we don't have to access the function, if not required
![Screenshot_2019-11-24_14-34-15](https://user-images.githubusercontent.com/32462195/69492330-88863900-0ec7-11ea-8e7c-5a995e89fd62.png)

Reply:
It's unnecessary check and will break if the endpoint is accessed using event_identifier